engine/src/object.cpp

```diff
@@ -5073,6 +5095,20 @@ bool MCObjectVisitor::OnBlock(MCBlock *p_block)
 	return true;
 }
 
+//////////
+
+bool MCObjectVisitor::IsRecursive()
```

Comment:
I wonder if these would be better off implemented as MCVisitStyle options? (Indeed, generalize MCVisitStyle to MCObjectVisitorOptions).